jekyll/_cci2/release/manage-releases.adoc

```diff
@@ -0,0 +1,158 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= Manage releases - Open preview
+:page-layout: classic-docs
+:page-liquid:
+:page-description: How to guide for managing releases from the CircleCI web app
+:icons: font
+:experimental:
+
+NOTE: CircleCI releases is currently in **open preview**. During open preview release features are available for free. Charges will be associated with release features once they are generally available.
+
+The steps outlined on this page guide you through the tasks required to manage your releases.
+
+[#prerequisites]
+== Prerequisites
+
+Before following the steps below, ensure the release environment has been set up and configured successfully. Refer to the xref:set-up-a-release-environment#[Set up a release environment] and xref:configure-your-kubernetes-component#[Configure your Kubernetes component] pages.
+
+[#restore-a-version]
+== Restore a version
+
+The _restore a version_ command restores a previously released version of a component.
+
+image::../../img/docs/releases/restore-a-version.png[Screenshot showing the restore a version button]
+
+[#restore-steps]
+=== Steps
+
+. Select btn:[Releases] in the CircleCI web app sidebar
+. You are now in the timeline view. Select your component from the dropdown menu.
+. Scroll to locate the version your want to restore, and click the restore icon in the **Actions** column
+. Read through the recommendations in the popup, and if you want to proceed type `RESTORE`, then click btn:[Restore]
+
+NOTE: Revert the associated changes in your source repository as soon as possible after using this command
+
+[#restore-notes]
+=== Notes
+
+Executing the _restore a version_ command from the CircleCI web app is the equivalent of using the following:
+
+* If you are using xref:configure-your-kubernetes-components#helm-rollback[Helm]:
++
+[,shell]
+----
+helm rollback <release-name> <revision-number> -n <namespace>
+----
+* If you are using Argo Rollouts and **not** Helm:
++
+[,shell]
+----
+kubectl argo rollouts undo <rollout-name> --to-revision=<revision_number>
+----
++
+If `revision_number` is not supplied the component will be rolled back to the previous revision
+* For a Kubernetes Deployment:
++
+[,shell]
+----
+kubectl rollout undo deployment/<deployment-name> -n <namespace> --to-revision=<revision-number>`
+----
+
+
+[#scale-a-version]
+== Scale a version
+
+The _scale a version_ command allows you to scale up or down a currently live version of a component.
```

Comment:
~ Question: Should this match our permissions doc? I think we just say "scale version", "restore version", etc. there - the audit log events would be similar as well.

Reply:
Yes! No idea why I have it like this now 🙃 I'll update. Thanks